apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -60,57 +60,59 @@ const styles = {
   },
 };
 
-function typeFormatter(type) {
+const typeFormatter = (type) => {
   return PROJECT_TYPE_MAP[type];
-}
+};
```

Comment:
Nit: There's no need to convert this or the other static 'formatter' functions to arrow functions.  Since they don't depend on the value of `this` we're not really getting anything out of using an arrow function here.